### PR TITLE
Transfers working

### PR DESCRIPTION
**Pull request by RalphLipe**
Various fixes to Jacoby Transfers and new test files.  Basic transfer logic working.

### CODE REVIEW

File: TricksterBots/Bots/Bridge/bridgebid/conventions/Stayman.cs [hunk 1]
```diff
@@ -183,6 +183,7 @@ private static bool InterpretResponderRebid(InterpretedBid rebid)
             if (rebid.declareBid.level == 3 && BridgeBot.IsMinor(rebid.declareBid.suit))
             {
                 //  (SAYC Booklet): if responder rebids three of either minor, he shows slam interest and at least a five-card suit
+                // TODO: Is this IFF declarer has "hit" the responder's suit?    Don't understand the logic here...
```
Comment (antross) [inline TricksterBots/Bots/Bridge/bridgebid/conventions/Stayman.cs:186]: So far, I haven't found where this logic originated from. If it doesn't make sense, it may have been a mistake (possibly copied from somewhere else when initially setting up the code). I'm okay replacing/removing it (and seeing what impact, if any, it has on the test suite).

**Review by antross — CHANGES_REQUESTED** [review]
Thanks again @RalphLipe! I've left some mostly minor feedback about keeping descriptions consistent, minimizing extra metadata in test files, and cleaning up some accidental whitespace/comments. After that this should be good to merge.

**Review by antross — APPROVED** [review]
